### PR TITLE
Add --main-branch argument to verify-copyright

### PR DESCRIPTION
**Pull request by KyleFromNVIDIA**

### CODE REVIEW

File: src/rapids_pre_commit_hooks/copyright.py [hunk 1]
```diff
@@ -125,14 +125,16 @@ def get_target_branch(repo, target_branch_arg=None):
       used. This allows GitHub Actions inside ``copy-pr-bot`` to easily use this tool.
     * If the configuration option ``rapidsai.baseBranch`` is defined, that branch is
       used. This allows users to locally set a base branch on a long-term basis.
```
Comment (bdice) [inline src/rapids_pre_commit_hooks/copyright.py:127]: Is `rapidsai.baseBranch` a git configuration option? I struggled to find how to set this earlier. Clarifying the docstring would help a lot.

File: src/rapids_pre_commit_hooks/copyright.py [hunk 2]
```diff
@@ -305,6 +309,11 @@ def main():
         "the --target-branch argument) to decide whether or not they need a copyright "
         "update."
     )
+    m.argparser.add_argument(
+        "--main-branch",
+        metavar="<main branch>",
+        help="main branch to use instead of branch-<major>.<minor>",
```
Comment (bdice) [inline src/rapids_pre_commit_hooks/copyright.py:315]: Let's clarify how `--main-branch` and `--target-branch` relate to one another in the docs. We should clarify that they effectively control the same thing, but with different precedence. Should it be legal to specify both of them?

Should we specify that `<major>` and `<minor>` are assumed to be two digits?

Comment (KyleFromNVIDIA) [reply src/rapids_pre_commit_hooks/copyright.py:319]: I've clarified the docs. It's legal to specify both `--main-branch` and `--target-branch`.

The current implementation does not assume `<major>` and `<minor>` are two digits, they can be any number of numerical digits long.

File: src/rapids_pre_commit_hooks/copyright.py [hunk 3]
```diff
@@ -303,7 +307,16 @@ def main():
         "Verify that all files have had their copyright notices updated. Each file "
         "will be compared against the target branch (determined automatically or with "
         "the --target-branch argument) to decide whether or not they need a copyright "
-        "update."
+        "update.\n\n"
+        "--main-branch and --target-branch effectively control the same thing, but "
+        "--target-branch has higher precedence and is meant only for a user-local "
+        "override, while --main-branch is a project-wide setting. Both --main-brnach "
```
Comment (bdice) [inline src/rapids_pre_commit_hooks/copyright.py:313]:
```suggestion
        "override, while --main-branch is a project-wide setting. Both --main-branch "
```

**Review by bdice — APPROVED** [review]
One typo. Otherwise LGTM!

**Comment by bdice** [comment]
@KyleFromNVIDIA You can merge this when you're ready.

**Pull request merged**